_opensearch/install/tar.md

```diff
@@ -7,144 +7,455 @@ nav_order: 50
-   ```
+Now that TLS certificates are installed and demo users were removed or assigned new passwords, the last step is to apply the configuration changes. This last configuration step requires invoking `securityadmin.sh` while OpenSearch is running on the host.
 
-   If you receive the `curl: (52) Empty reply from server` error, you are likely protecting your cluster with the security plugin and you need to provide credentials. Modify the following command to use your username and password:
+1. Start OpenSearch. It must be running for `securityadmin.sh` to apply changes.
+   ```bash
+   # Change directories
+   cd /path/to/opensearch-{{site.opensearch_version}}/bin
 
+   # Run the service in the foreground
+   ./opensearch
+   ```
+1. Open a separate terminal session with the host and navigate to the directory containing `securityadmin.sh`.
    ```bash
-   curl -XPOST https://localhost:9200/_plugins/_performanceanalyzer/cluster/config -H 'Content-Type: application/json' -d '{"enabled": true}' -u 'admin:admin' -k
+   # Change to the correct directory
+   cd /path/to/opensearch-{{site.opensearch_version}}/plugins/opensearch-security/tools
    ```
-
-1. Finally, enable the Root Cause Analyzer (RCA) framework
-
+1. Invoke the script. See [Apply changes using securityadmin.sh]({{site.url}}{{site.baseurl}}/security-plugin/configuration/security-admin/) for definitions of the arguments you must pass.
    ```bash
-   curl -XPOST localhost:9200/_plugins/_performanceanalyzer/rca/cluster/config -H 'Content-Type: application/json' -d '{"enabled": true}'
+   # You can omit the environment variable if you declared this in your $PATH.
+   OPENSEARCH_JAVA_HOME=/path/to/opensearch-{{site.opensearch_version}}/jdk ./securityadmin.sh -cd /path/to/opensearch-{{site.opensearch_version}}/config/opensearch-security/ -cacert /path/to/opensearch-{{site.opensearch_version}}/config/root-ca.pem -cert /path/to/opensearch-{{site.opensearch_version}}/config/admin.pem -key /path/to/opensearch-{{site.opensearch_version}}/config/admin-key.pem -icl -nhnv
    ```
 
-   Similar to step 4, if you run into `curl: (52) Empty reply from server`, run the command below to enable RCA
+### Verify that the service is running
 
-   ```bash
-   curl -XPOST https://localhost:9200/_plugins/_performanceanalyzer/rca/cluster/config -H 'Content-Type: application/json' -d '{"enabled": true}' -u 'admin:admin' -k
-   ```
+OpenSearch is now running on your host with custom TLS certificates and a secure user for basic authentication. The last step is verifying that the host is reachable by other clients.
 
-{% comment %}
+During previous tests you directed requests to `localhost`. Now that TLS certificates have been applied, and the new certificates reference your hosts actual DNS record, the CN check will fail and the certificate will be considered invalid if you send requests to `localhost`. Instead, requests should be sent to the address you specified while generating the certificate.
```

Comment:
Delete comma after "applied" (first sentence)

Reply:
Can we rewrite the second and last sentences to read: "Now that TLS certificates have been applied and the new certificates reference your hosts actual DNS record, if you send requests to the `localhost`, the CN check will fail and the certificate will be considered invalid. Instead, send requests to the address you specified while generating the certificate."

Reply:
Comma deleted.  `localhost` is a networking concept - not the same as saying "local host" - so saying "the `localhost`" wouldn't make sense.

Reply:
Oh I think I see what you added.  Sorry took me a second to figure out what you changed.

Reply:
Changes applied!  Thank you!